protyping/auto-segmentation/storage_and_inference_scp.py

```diff
@@ -39,20 +39,20 @@ def handle_accepted(event):
 def handle_store(event):
     """Handle EVT_C_STORE events."""
 
-    ds = event.dataset
-    ds.file_meta = event.file_meta
+    with lock:
```

Comment:
Hehe, you any this in anyway :). Sure... It's not exactly a documentation update though...

Reply:
Oh woops was this included in the docs PR? I was sure i checked out a new branch for this commit :confused: